prime-router/docs/api/waters-delivery.yml

```diff
@@ -0,0 +1,86 @@
+openapi: "3.0.3"
+info:
+  title: ReportStream Delivery API
+  description: This API reflects the point of view of the STLTs and provides lookup functionality
+  version: "1.0"
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://open.cdc.gov
+    email: usds@cdc.gov
+  license:
+    name: Creative Commons Zero v1.0 Universal
+    url: https://github.com/CDCgov/prime-reportstream/blob/master/LICENSE
+servers:
+  - url: https://prime.cdc.gov/api
+    description: Production server (uses live data)
+  - url: http://localhost:7071/api
+    description: Developer Environment
+paths:
+  /api/waters/{organization}/deliveries:
+    get:
+      operationId: getAvailableReports
+      summary: Get available reports for a STLT
+      description: Retrieve a list of reports available for download by a STLT
+      security:
+        - OktaAuth: [ ANONYMOUS ]
+        - Server2Server: []
+      parameters:
+        - name: organization
+          in: path
+          description: receiver whose reports we want to see
+          required: true
+          schema:
+            type: string
+        - $ref: "./components/sort_and_filter.yml#/parameters/sortOrder"
```

Comment:
Splitting these into reusable references makes it easier for use to maintain consistency, but it may make it unusable to customers without jumping through hoops to dereference or installing a local openapi editor and copying the whole spec locally.

My feeling is that we should make these easy to read and use on-line.  It may be feasible to resolve this by using a URL reference like: `$ref: https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/components/sort_and_filter.yml#/parameters/sortOrder`

Reply:
This seems to work when I'm trying it locally. If you wanna replicate it you do need to change `master` to the latest commit id